tests/Api/Responses/shop/paying_for_orders_by_apple_pay/test_paying_with_a_valid_apple_pay_token_for_an_order.json

```diff
@@ -0,0 +1,12 @@
+{
+  "@context": {
+    "@vocab": "http://localhost/api/v2/docs.jsonld#",
+    "hydra": "http://www.w3.org/ns/hydra/core#",
+    "status": "PayResult/status",
+    "transactionPaymentUrl": "PayResult/transactionPaymentUrl"
+  },
+  "@type": "PayResult",
+  "@id": "/api/v2/.well-known/genid/@string@",
+  "status": "pending",
+  "transactionPaymentUrl": "apple_pay://@string@"
```

Comment:
It helps us to make sure we got a response for Apple Pay from Mockoon; we can add something similar for other payment methods in a separate PR